README.md

```diff
@@ -1,151 +1,11 @@
-The purpose of the corefx-testdata repo is to store corefx test file dependencies.
+# CoreFx-TestData
 
-Large test inputs (eg. over 1MB) should not bloat the corefx repo. Instead, they have their own dedicated repo, and the corefx build process pulls them via MyGet. The advantage of this is that their bloated history is not stored locally.
+This repository contains test assets that are binary file or too large to be checked into corefx directly. Packages are produced and uploaded to the dotnet blob feed (https://dotnetfeed.blob.core.windows.net/dotnet-core/index.json).
 
-Steps to add test inputs to this repo:
+Uploading the packages is currently a manual step. Community members can submit PRs but the deployment needs to be done by a .NET team member as described below.
 
-1. Clone the corefx-testdata repo.
-
-2. In the root folder of the corefx-testdata repo, locate the folder with the namespace of the tests you are working on and place your input files inside this folder in the appropriate location.
-
-### **Example**
-
-Let's say you are adding tests in the ````System.IO.Compression```` namespace for the GZip features. You can save your new file called example.gz inside the folder:
-
-````corefx-testdata/System.IO.Compression.TestData/GZipTestData/````
- 
- 3. In the root folder of the corefx-testdata repo, locate the *.nuspec file for your namespace, then:
-
-    a) Bump the version by one.
-
-    b) Add a new file entry for each one of your new files, where src points to the relative path inside corefx-testdata and target points to the location inside NuGet (simply substitute the namespace folder with "content").
-
-### **Example**
-
-Open the file ```System.IO.Compression.TestData.nuspec```.
-
-Locate the ````<version>```` element. If the current version is:
-
-````xml
-<version>1.0.7-prerelease</version>
-````
-
-then bump it to:
-
-````xml
-<version>1.0.8-prerelease</version>
```

Comment:
What's the versioning story now?

Reply:
I just added documentation for that and bumped all package versions to get into a clean state with prerelease labels.